kubernetes/istio-rbac/istio-rbac.yaml

```diff
@@ -97,7 +97,22 @@ metadata:
   name: istio-sidecar-role-binding
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: details-service-account
```

Comment:
I mean: the service accounts and bindings are only specific to the bookinfo sample, should not be installed/used if istio is installed for a different app.